configurations/dockerDataTemplate.go

```diff
@@ -9,10 +10,10 @@ type instruction interface {
 	WriteInstruction() string
 }
 
-type serverConfig []instruction
+type config []instruction
 
-// Decodes the yaml data and gives the serverConfig instance having all the dockerfile instructions.
-func (s *serverConfig) UnmarshalYAML(unmarshal func(interface{}) error) error {
+// Decodes the yaml data and gives the config instance having all the dockerfile instructions.
```

Comment:
// UnmarshalYAML decodes the yaml configuration to obtain dockerfile instructions.

Reply:
Okay